packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx

```diff
@@ -118,10 +154,11 @@ const ActiveVisitsTable = () => {
       }
     }
 
-    return activeVisits;
-  }, [searchString, activeVisits]);
+    return activeVisits.map(transformVisitForDisplay);
```

Comment:
Why don't we transform the activeVisits before the searchResults itself?
Also, the check above needs to be
```
`${value}`.toLocaleLowerCase().includes(search.toLocaleLowerCase())

Reply:
`search` is already assigned to `searchString.toLowerCase()`